abcd/frontends/commandline/commands.py

```diff
@@ -4,39 +4,46 @@
 import numpy as np
 
 from abcd.frontends.commandline.decorators import check_remote, init_config, init_db
+from abcd.backends.atoms_opensearch import OpenSearchDatabase
 
 logger = logging.getLogger(__name__)
 
 
 @init_config
-def login(*, config, name, url, **kwargs):
+def login(*, config, name, url, disable_ssl=False, **kwargs):
     logger.info(
-        'login args: \nconfig:{}, name:{}, url:{}, kwargs:{}'.format(config, name, url, kwargs))
+        "login args: \nconfig:{}, name:{}, url:{}, kwargs:{}".format(
+            config, name, url, kwargs
+        )
+    )
     from abcd import ABCD
 
-    db = ABCD.from_url(url=url)
+    db = ABCD.from_url(url=url, use_ssl=(not disable_ssl))
     info = db.info()
 
-    config['url'] = url
+    config["url"] = url
+    config["use_ssl"] = not disable_ssl
```

Comment:
Something I learned recently is that you can do:
```suggestion
    config.update(url=url, use_ssl=not disable_ssl)
```